stylesheet/main.css

```diff
@@ -97,6 +97,47 @@ body {
     font-size: 18px;
     text-transform: uppercase;
 }
+
+.btnContainer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 10vh;
+}
+
+.btn {
+    position: relative;
+}
+
+.btn > img {
+    position: relative;
+    height: 8vh;
+}
```

Comment:
`>` is a direct child selector but I don't see any direct child of `.btn`that is a `img`

Reply:
I've updated this